geth-utils/gethutil/trace.go

```diff
@@ -20,6 +20,7 @@ import (
 // while replaying a transaction in debug mode as well as transaction
 // execution status, the amount of gas used and the return value
 type ExecutionResult struct {
+	Invalid     bool           `json:"invalid"`
```

Comment:
`Invalid` is not a field in github.com/ethereum/go-ethereum/internal/ethapi.ExecutionResult

Reply:
That's okay, this is mainly only there to make things a bit cleaner and to handle error messages better. If we run a trace against a real node and it errors our or returns nothing we'll also know it's invalid.